tests/test_bioimageio_spec_version.py

```diff
@@ -29,8 +29,8 @@ def test_bioimageio_spec_version(mamba_cmd: Optional[str]):
     # get currently pinned bioimageio.spec version
     meta = metadata("bioimageio.core")
     req = meta["Requires-Dist"]
-    assert req.startswith("bioimageio.spec ==")
-    spec_ver = req[len("bioimageio.spec ==") :]
+    assert req.startswith("bioimageio.spec==")
```

Comment:
why does this need to change?
there is a space in https://github.com/bioimage-io/core-bioimage-io-python/blob/66665c573b06a43ae5a0e17d81961619235a66b2/setup.py#L32

Reply:
When I cloned the repo and run the tests, it was the only test that was failing due to this space. Maybe I am missing something. Also I am not sure why the CI don't trigger the tests for this PR.

Reply:
interesting. I enable CI now for this PR